cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -75,8 +61,8 @@ <h1>Explore interest rates</h1>
                     <div class="content-l_col content-l_col-1-3">
                         <div class="rc-illu-inner">
                             <img class ="rc-home-illu"
-                                 src="{{ static('apps/owning-a-home/img/ill-chart.png') }}"
-                                 alt="Illustration of interest rate chart">
+                                    src="{{ static('apps/owning-a-home/img/ill-chart.png') }}"
+                                    alt="Illustration of interest rate chart">
```

Comment:
There are a whole bunch of indentation changes like this that are causing misalignment of attributes over multiple lines. Some kind of editor automation issue, perhaps?

Reply:
Hopefully I got them all in https://github.com/cfpb/consumerfinance.gov/pull/6446/commits/8f338455c1b8be707350340de0368f5865ffe5a4

Reply:
And https://github.com/cfpb/consumerfinance.gov/pull/6446/commits/d208ae3ed04e1ec25f3b948168d3197990a0ae75